docs/pages/community.md

```diff
@@ -47,6 +47,8 @@ In addition to the [official][official] [tutorials][tutorials], here are Remix t
 
 - [How to Debug Remix loaders and actions in VS Code][how-to-debug-remix-loaders-and-actions-in-vs-code] by [Michael Carter][michael-carter].
 - [How to build a Remix website with Sanity.io and live preview][how-to-build-a-remix-website-with-sanity-io-and-live-preview] by [Simeon Griggs][simeon-griggs]
+- [Build A Pet Management System With Remix, Prisma, and Postgres](build-a-pet-management-system-with-remix-prisma-and-postgres
+) by [Austin Gil]
```

Comment:
```suggestion
- [Build A Pet Management System With Remix, Prisma, and Postgres][build-a-pet-management-system-with-remix-prisma-and-postgres]
  by [Austin Gil][austin-gil]
```

Reply:
Thanks. I've never done links like that before.

Reply:
Our CI takes care of these, so you could just use normal links as well in future PRs.